maquetado/index.html

```diff
@@ -8,70 +8,75 @@
     <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 <body>
+  <div class="navbar">
+    <img class="nav-logo" src="./assets/wolox_logo.svg" alt="">
+    <p class="nav-text">Developer Tools</p>
+  </div>
   <div class="content-title">
-    <img src="./assets/triangle.svg" alt="" class="logo-title">
+    <img src="./assets/triangle.svg" class="logo-title">
```

Comment:
This is a decorative image... It should have the `alt` attribute but empty.
Remember W3C rules:
_Guidelines for the alt text:_
- The text should describe the image if the image contains information
- The text should explain where the link goes if the image is inside an <a> element
- Use alt=“” if the image is only for decoration (edited)